mmv1/products/vertexai/product.yaml

```
@@ -17,9 +17,9 @@ display_name: Vertex AI
 versions:
   - !ruby/object:Api::Product::Version
     name: ga
-    base_url: https://{{region}}-aiplatform.googleapis.com/v1/
+    base_url: https://aiplatform.googleapis.com/v1/
```

`{{region}}-` is correct for the provider resources. you just need to override the cai_base_url, similar to https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/products/cloudrun/product.yaml

```suggestion
    base_url: https://{{region}}-aiplatform.googleapis.com/v1/
    cai_base_url: https://aiplatform.googleapis.com/v1/
```